src/app/home/home.page.html

```diff
@@ -1,4 +1,4 @@
-<ion-content>
+<ion-content class="home-page">
```

Comment:
in generale la dove possiamo usare l'html markup usiamolo non c'e' bisogno di definire una classe basta scrivere ion-content